src/templates/article.js

```diff
@@ -179,6 +182,17 @@ const Article = props => {
                 <meta property="og:title" content={articleTitle}></meta>
                 <meta property="og:url" content={articleUrl}></meta>
                 <link rel="canonical" href={articleUrl}></link>
+                <script type="application/ld+json">
+                    {getPageSchema(articleUrl, {
+                        articleBody: __refDocMapping.source,
```

Comment:
This is pretty-raw content from the rST, but not sure if there is something better we can use here. 

```
:template: devhub-article\n\n.. ---------------------------------------------------------------\n.. META FIELDS FOR SEO / SOCIAL\n.. ---------------------------------------------------------------\n\n.. meta-description:: \n\n   This post will begin by describing the database capabilities required by modern multi-data center applications.\n\n.. twitter::\n   :site: @mongodb\n   :creator: @jayrunkel\n   :title: Active-Active Application Architectures with MongoDB\n   :image: https://webassets.mongodb.com/_com_assets/cms/image4-3d593nantp.png\n\n   This post will begin by describing the database capabilities required by modern multi-data center applications\n\n.. ---------------------------------------------------------------\n.. ARTICLE METADATA FIELDS (TO POPULATE FILTERS AND ARTICLE PAGE)\n.. ---------------------------------------------------------------\n\n.. include:: /includes/authors/runkel-jay.rst\n\n.. pubdate:: 2017-12-08\n\n.. type:: article\n\n.. level:: advanced\n\n.. tags:: \n\n   * Technical\n\n.. products::\n\n   * MongoDB\n\n.. atf-image:: /images/atf-images/generic/purple.png\n\n.. ---------------------------------------------------------------\n.. ARTICLE CONTENT\n.. ---------------------------------------------------------------\n\n====================================================\nActive-Active Application Architectures with MongoDB\n====================================================\n\n.. introduction::\n\n    Introduction\n    ------------\n\n    Determining the best database for a modern application to be deployed\n    across multiple data centers requires careful evaluation to accommodate\n    a variety of complex application requirements. The database will be\n    responsible for processing reads and writes in multiple geographies,\n    replicating changes among them, and providing the highest possible\n    availability, consistency, and durability guarantees. But not all\n    technology choices... (rest of article)
```

Reply:
Not sure, still waiting for the final schema from marketing, so maybe they won't want the whole thing.